run-tests.php

```diff
@@ -2160,22 +2160,42 @@ function run_test($php, $file, $env)
 				return 'SKIPPED';
 			}
 
+			$keywordFound = false;
+
 			if (!strncasecmp('info', ltrim($output), 4)) {
 				if (preg_match('/^\s*info\s*(.+)\s*/i', $output, $m)) {
 					$info = " (info: $m[1])";
+					$keywordFound = true;
 				}
 			}
 
 			if (!strncasecmp('warn', ltrim($output), 4)) {
 				if (preg_match('/^\s*warn\s*(.+)\s*/i', $output, $m)) {
 					$warn = true; /* only if there is a reason */
 					$info = " (warn: $m[1])";
+					$keywordFound = true;
 				}
 			}
 
 			if (!strncasecmp('xfail', ltrim($output), 5)) {
 				// Pretend we have an XFAIL section
 				$section_text['XFAIL'] = trim(substr(ltrim($output), 5));
+				$keywordFound = true;
+			}
+
+			if (!$keywordFound && trim($output) !== '') {
+				$bork_info = trim($output);
+				show_result("BORK", $bork_info, $tested_file, 'reason: invalid output from SKIPIF', $temp_filenames);
+				$PHP_FAILED_TESTS['BORKED'][] = array(
```

Comment:
We can use the new syntax here for arrays

```suggestion
				$PHP_FAILED_TESTS['BORKED'][] = [
```

Reply:
We can, but we can also be consistent and replace the old syntax in the whole file not just this occurrence.